### DISCUSSION TITLE
Cannot select project license

### DISCUSSION THREAD

**Discussion by CharacterNotFound** [modrinth/.github#298] [Other discussions]
For some reason, when creating a project, I simply do not have a list of licenses on this page. I could continue without a license, but in order to submit it for moderation, I must indicate it. How to solve this problem?
![Screenshot_9](https://github.com/user-attachments/assets/6379fb94-191e-46e8-8ccd-d1b9278b5cf8)

**Comment by maideniles** [comment]
I have the very same problem. There is no option to select a license for a project, and it won't let me submit a project without 'choosing' one.

**Comment by Prospector** [comment]
Should no longer be an issue
